infra_event_notifier/cli/terragrunt.py

```diff
@@ -18,17 +19,21 @@
     from typing_extensions import override
 
 from yaml import SafeLoader, load
+
 from infra_event_notifier.cli.command import (
     BaseCommand,
     Subparsers,
     add_dryrun,
 )
 from infra_event_notifier.backends import datadog
+from infra_event_notifier.cli.util import workspace_root
 
 TERRAGRUNT_EVENT_SOURCE = "terragrunt"
 
 
 class SentryKubeConfig:
+    DEFAULT_CONFIG = "cli_config/configuration.yaml"
```

Comment:
I don't think we should depend on the sentry-kube config in this library. 
That is meant for the sole usage of sentry-kube, sentry-kube has the abstractions to manage the config and the format of that config can and does change. 
Another library depending on it is likely to break in those scenarios. 

It seems to me you are depending only on the region name. 
Any reason you cannot simply use this `region = tgslice.split("/")[-1]` ?

Reply:
Because this introduces an inconsistency with the rest of our datadog logs, which is the reason this logic exists. I can tell it's something people care about because we actively patch the results of it: see #15, which was authored in ops and then copied to sentry-infra-tools

We *can* opt to change this behavior, and I would honestly love to. But it will need to be a broader conversation of what we expect to see in Datadog.

Reply:
It is important to have the right region names in datadog as people primarily filter the events dashboard by region. 

Ideally we should have some shared basic config every tool has access to. 
What about this:
- Provide such simpler config file to this script (A simple mapping from terragrunt region to sentry region).
- Add the file at the root of terragrunt in the ops repo
- pass it to the tool.

At least we make it terragrunt specific for now (as the mapping is terragrunt specific) rather than introducing an implict dependency on the sentry-kube config.

Reply:
Went with approach 1